app/forms/declaration/TotalNumberOfItems.scala

```diff
@@ -50,6 +55,14 @@ object TotalNumberOfItems extends DeclarationPage {
   val removeCommasFirst = (validator: String => Boolean) => (input: String) => validator(input.replaceAll(",", ""))
   val notJustCommas = (input: String) => !input.forall(_.equals(','))
 
+  val equalsIgnoreCaseOptionString = (value: String) => (input: Option[String]) => input.map(_.equalsIgnoreCase(value)).getOrElse(false)
+  val isEmptyOptionString = (input: Option[String]) => isEmpty(input.getOrElse(""))
+  val isAlphabeticOptionString = (input: Option[String]) => isAlphabetic(input.getOrElse(""))
+  val lengthInRangeOptionString = (min: Int) => (max: Int) => (input: Option[String]) => lengthInRange(min)(max)(input.getOrElse(""))
+
+  def isFieldEmpty(field: String): Condition = _.get(field).map(_.isEmpty()).getOrElse(true)
+  def isFieldNotEmpty(field: String): Condition = _.get(field).map(!_.isEmpty()).getOrElse(false)
```

Comment:
Would it be worth to move these validation methods (lines 58-64) to `FieldValidator.scala` ?